reproducibility_project/src/engines/mcccs/project.py

```diff
@@ -165,59 +246,130 @@ def cool_finished(job):
 @Project.label
 def equil_finished(job):
     """Check if equil stage is finished."""
-    step = "equil"
+    try:
+
+        step = "equil" + str(job.doc.equil_replicates_done - 1)
+    except (KeyError, AttributeError):
+        step = "equil" + "0"
+
     run_file = job.ws + "/run.{}".format(step)
     if job.isfile("run.{}".format(step)):
         with open(run_file) as myfile:
             if "Program ended" in myfile.read():
                 return True
             else:
                 return False
+    else:
+        return False
+
+
+def sanitize_npt_log(step):
+    """Sanitize the output logs for NPT simulations."""
+    import numpy as np
+
+    files = glob("fort*12*{}*".format(step))
+    arrays = []
+    for filecurrent in files:
+        array = np.genfromtxt(filecurrent, skip_header=1)
+        arrays.append(array)
+    arrays = np.vstack(arrays)
+    arrays[:, 0] = arrays[:, 0] / 10
```

Comment:
Lets add a comment here mentioning what these units are being converted to, or use `unyt`

Reply:
Thanks.